packages/suite/src/components/suite/DeviceDisplay/parseTextToPagesAndLines.ts

```diff
@@ -52,10 +53,26 @@ export const parseTextToPagesAndLines = ({
     deviceModel,
     text,
 }: ParseTextToLinesParams): ParseTextToLinesResult => {
+    const getCharsPerRow = (): number => {
+        // T1B1 has varying row length depending on address type.
+        if (deviceModel === DeviceModelInternal.T1B1) {
+            switch (text.length) {
+                // legacy and legacy segwit accounts:
+                case 34:
+                    return 17;
+                // legacy segwit testnet accounts:
```

Comment:
I suggest to rewrite it like this: 

```ts
const isLegacySegwitAccount = text.length === 34

if (isLegacySegwitAccount) { return 17}

const isLegacySegwitTestnetAccount = text.length === 35

if (isLegacySegwitTestnetAccount) { return 18 }
```

This way, we dont need comments and code is self-explanatory

Reply:
Switch is fine, but the comments that can be replaced by proper variable naming aren't :D